gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspAATask.kt

```diff
@@ -404,14 +404,14 @@ abstract class KspGradleConfig @Inject constructor() {
     abstract val libraries: ConfigurableFileCollection
 
     @get:Internal
-    abstract val jdkHome: Property<File>
+    abstract val jdkHome: DirectoryProperty
```

Comment:
Breaking change related to https://github.com/google/ksp/pull/2459#discussion_r2480115204.